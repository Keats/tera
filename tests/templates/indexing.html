{{ reviews.1.title }}
{{ a_tuple.0 }}
{% for u in an_array_of_tuple %}{{ u.1 }}{% endfor %}
