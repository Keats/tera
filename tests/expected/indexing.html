My review
1
22